pyhomematic/devicetypes/actors.py

```diff
@@ -753,7 +755,7 @@ def __init__(self, device_description, proxy, resolveparamsets=False):
 
     @property
     def ELEMENT(self):
-        return [3, 7]
+        return [2, 3, 4, 6, 7, 8]
```

Comment:
Why do you change this? Neither HmIP-PCBS2 nor HmIP-FSI16 (the devices this PR is about) make use of the `IPMultiIO` class. 🤔 
If this is a fix because HmIP-MIOB is not correctly implemented, please add an explanation.

Reply:
Nevermind, now I see why.

Reply:
This is a fix.
I Noticed for the HmIP-MIOB in my case the "Kühlmnodus" is only working on Channel 6.
Based on CCU3 UI there are 6 Switches on channel 2 till 4 and 6 till 8.
So I added the switches.
I tested this in my environment. I was able to hear on each channel the click of the relais.
In my case there is currently ony channel 6 used.

The other changes are for the support for HmIP-PCBS2 and HmIP-FSI16